### PR TITLE
Fix/restore password error messages

### PR DESCRIPTION
**Pull request by AKutusenko**
## Describe your changes

- I have changed error messages

## Issue ticket code (and/or) and link

- [Link to JIRA ticket](https://testhomework4.atlassian.net/jira/software/projects/AG/boards/6?assignee=5fd26b3708757f006e4d61d4&selectedIssue=AG-148)
- [Link to JIRA ticket](https://testhomework4.atlassian.net/jira/software/projects/AG/boards/6?assignee=5fd26b3708757f006e4d61d4&selectedIssue=AG-150)

### **General**

- [x] Assigned myself to the PR
- [x] Assigned the appropriate labels to the PR
- [x] Assigned the appropriate reviewers to the PR
- [ ] Updated the documentation
- [x] Performed a self-review of my code
- [x] Types for input and output parameters
- [x] Don't have "any" on my code
- [x] Used the try/catch pattern for error handling
- [x] Don't have magic numbers
- [x] Compare only with constants not with strings
- [x] No ternary operator inside the ternary operator
- [x] Don't have commented code
- [x] no links in the code, env links should be in env file (for example: server url), constant links (for example default avatar URL) should be in constant file.
- [x] Used camelCase for variables and functions
- [x] Date and time formats are on the constants
- [x] Functions are public only if it's used outside the class
- [x] No hardcoded values
- [ ] covered by tests
- [x] Check your commit messages meet the [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/).

### Frontend

- [x] Components and business logic are separated
- [x] Colors, Font Size, and Font Name is on the theme or in the constants
- [x] No text in the components, use i18n approach
- [x] No inline styles
- [ ] Imports are absolute
- [ ] Attach a screenshot if PR has visual changes.

### CODE REVIEW

**Review by rudenkoserhii — APPROVED** [review]

File: src/sections/restore-password-view/form.tsx [hunk 1]
```diff
@@ -49,15 +49,17 @@ export default function RestorePasswordForm(): JSX.Element {
...
 
       return error;
+    } finally {
+      setActiveRequestsCount((prevCount) => prevCount - 1);
     }
   });
 
   return (
     <>
-      {(isSubmitting || isLoading) && (
```
Comment (lyudad) [inline src/sections/restore-password-view/form.tsx:60]: а куда ты лоадер всунул? или где лоадер обрабатывается, не нашла

Comment (AKutusenko) [reply src/sections/restore-password-view/form.tsx:60]: на 62 строке

Comment (AKutusenko) [reply src/sections/restore-password-view/form.tsx:60]: я поменял уловие для лоадера так он плавнее работает

Comment (lyudad) [reply src/sections/restore-password-view/form.tsx:60]: но activeRequestsCount не остлеживает лоадинг данных

Comment (AKutusenko) [reply src/sections/restore-password-view/form.tsx:60]: не мигает между сабмитом и лоадингом

Comment (AKutusenko) [reply src/sections/restore-password-view/form.tsx:60]: Он отслеживает когда весь блок try выполнился и тогда он обнуляется и модалка с лоадером пропадает
так плавнее работает и не мигает между isSubmit и isLoading

**Review by lyudad — APPROVED** [review]

**Pull request merged**